bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx

```diff
@@ -28,21 +38,31 @@ export default class Whiteboard extends React.Component {
           transitionLeaveTimeout={400}
         >
           <svg
-            viewBox={ '0 0 ' + this.props.current_slide.slide.width + ' ' + this.props.current_slide.slide.height}
+            viewBox={ x + ' ' + y + ' ' + viewBoxWidth + ' ' + viewBoxHeight}
```

Comment:
Yes, it's working perfectly fine. Let's use this method instead of concatenation then. :+1: